build-tools/makefile_components/base_container_arm64.mak

```diff
@@ -0,0 +1,28 @@
+# Container section of standard makefile (arm64)
```

Comment:
The stuff in build-tools is not to be edited here, it's essentially an include. 

That said, DDEV-Local is now the only relevant consumer of build-tools, so it's my opinion that we should move the piece-parts  that we need out of build-tools/makefile_components and into the Makefile.